pkg/openstack/common.go

```diff
@@ -290,6 +291,22 @@ func EnsureEndpointConfig(
 					} else if (ctrlResult != ctrl.Result{}) {
 						return endpoints, ctrlResult, nil
 					}
+					// Delete the issued certificate if it exists
+					certName := fmt.Sprintf("%s-route", ed.Name)
+					cert := certmanager.NewCertificate(
+						&certmgrv1.Certificate{
+							ObjectMeta: metav1.ObjectMeta{
+								Name:      certName,
+								Namespace: ed.Namespace,
+							},
+						},
+						5*time.Second,
+					)
+					err = cert.Delete(ctx, helper)
```

Comment:
how about the cert secret? can not remember what we said to do with it in this situation? delete it? add the owner to the secret?

Reply:
iirc we agreed that we keep it as it is, and if user wants to delete secret with the certs he can set the `--enable-certificate-owner-ref` in cert-manager, and we only document it.